src/components/ForgotPassword.tsx

```diff
@@ -0,0 +1,51 @@
+import { useRef, useState } from "react";
+import { checkValidEmail } from "./utils/validate";
+import HeaderComponent from "./ui/HeaderComponent";
+import { Link } from "react-router-dom";
+
+const ForgotPassword: React.FC = () => {
+  const [errorMessage, setIsErrorMessage] = useState<string | null>(null);
+  const email = useRef<HTMLInputElement>(null);
+
+  function handleButtonClick() {
+    if (email.current) {
+      console.log("email = ", email.current.value);
+      const message = checkValidEmail(email.current.value);
+      setIsErrorMessage(message);
+    }
+  }
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+  };
+
+  return (
+    <div className="absolute">
+      <HeaderComponent />
+      <form
+        onSubmit={handleSubmit}
+        className="absolute w-1/4 p-12 text-white bg-black top-0 mx-auto left-0 right-0 my-44 rounded-lg bg-opacity-85 "
+      >
+        <p className="text-3xl pb-4 pl-2 font-semibold">Forgot Password?</p>
```

Comment:
Can use HTML entities for symbols  are much preferred as it displays the character accurately.

Reply:
updated with new name